python_benchmarks/test_softmax_fwd.py

```diff
@@ -87,8 +95,10 @@ def test_softmax_fwd_baseline_benchmark(
 ):
     clear_cuda_cache()
     input = torch.randn(size, device="cuda", dtype=dtype)
+
     run_benchmark(
         benchmark,
         torch.compile(softmax_fwd_fn) if compile else softmax_fwd_fn,
         [input, reduction_axis],
+        iobytes=softmax_fwd_iobytes(size, dtype),
```

Comment:
@jacobhinkle I am using the IO bytes computation to pass the IO bytes for the benchmark. This will skip the IO bytes computation and use this value instead.